app/views/notifications/index.html.erb

```diff
@@ -25,9 +25,14 @@
    *
    * @param {Event} The click event on the notification.
    */
-  function trackNotification(e) {
-    let title = e.target.parentElement.id
-    ahoy.track("Clicked Welcome Notification", { title });
+  function trackNotification(event) {
+    let title = event.target.parentElement.id;
+    let text = event.target.text;
```

Comment:
Destructuring ftw. The syntax `id: title` below allows you to rename `id` to `title`.

```suggestion
    const { parentElement: { id: title }, text } = event.target;
```

Reply:
love it! been doing too much ruby recently and apparently forgot that i can destructure, lol 😂